libs/ui/src/precinct_scanner_polls_report.tsx

```diff
@@ -63,6 +63,7 @@ const SignatureLine = styled.div`
   width: 50%;
   min-height: 4em;
   &::before {
+    font-family: 'Noto Emoji', sans-serif;
```

Comment:
This may be overly specific and un-necessary… but I don't know how to emulate a Poll Worker card with a polls closed report on it.